packages/rocketchat-reactions/setReaction.js

```diff
@@ -1,7 +1,12 @@
 /* globals msgStream */
 Meteor.methods({
-	setReaction(reaction, messageId) {
-		if (!Meteor.userId()) {
+	setReaction(reaction, messageId, user) {
```

Comment:
Please revert this change and instead run this method as the user who is doing the reacting like such:

```javascript
Meteor.runAsUser(user._id, () => {
	Meteor.call('setReaction', reaction, messageId);
});
```

Otherwise this is a breaking change for all of the clients out there who call this method as the user and don't pass in the user object.